### PR TITLE
Remove the emphasised tags and updated the placeholder text

### LINKED ISSUE #20890: Remove `<em>` tags from code examples
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

All code examples that include `<em>` tags to emphasize text. For example:

````
```shell
git init <em>YOUR_REPO</em>
```
````

### What part(s) of the article would you like to see updated?

The `em` tag does not render correctly if you use a language other than `shell` or if you include a `copy` button. We have updated our [style guide](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#code-sample-syntax-highlighting) to instruct contributors to use all uppercase instead of `em` tags to indicate placeholder text.

You should update code examples to:
1. Ensure that any text currently within `em` tags is all uppercase
2. Remove the `em` tags

For example:

````
```shell
git init <em>your_repo</em>
```
````

will become

````
```shell
git init YOUR_REPO
```
````

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #20890 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

- As per the style guide, removed the `<em>` tags from the code examples of all the pages.
- Capitalized the placeholder text and updated the text while maintaining consistency throughout the doc pages.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/repositories/creating-and-managing-repositories/cloning-a-repository.md [hunk 1]
```diff
@@ -42,14 +42,15 @@ You can clone your existing repository or clone another person's existing reposi
...
+gh repo clone REPOSITORY
 ```
 
 You can also use the GitHub URL to clone a repository.
 
 ```shell
-gh repo clone <em>https://github.com/cli/cli</em>
+gh repo clone URL
 ```
+Replace URL with the URL of a GitHub repository. For example, `https://github.com/cli/cli`
```
Comment (ethanpalm) [inline content/repositories/creating-and-managing-repositories/cloning-a-repository.md:53]:
```suggestion
```

File: content/repositories/creating-and-managing-repositories/cloning-a-repository.md [hunk 2]
```diff
@@ -42,14 +42,15 @@ You can clone your existing repository or clone another person's existing reposi
...
 ```shell
-gh repo clone <em>repository</em>
+gh repo clone REPOSITORY
 ```
 
 You can also use the GitHub URL to clone a repository.
 
 ```shell
-gh repo clone <em>https://github.com/cli/cli</em>
+gh repo clone URL
```
Comment (ethanpalm) [inline content/repositories/creating-and-managing-repositories/cloning-a-repository.md:51]:
```suggestion
gh repo clone https://github.com/PATH-TO/REPOSITORY
```

**Review by ethanpalm — DISMISSED** [review]
Thanks for creating this PR, @codewithdev! ✨ 🚀 

I'm just going to push a few small changes and then get this merged once all the checks are passing.

**Review by ethanpalm — DISMISSED** [review]

File: content/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ For example:
 {% windows %}
 
 ```shell
-run.cmd --check --url <em>https://github.com/octo-org/octo-repo</em> --pat <em>ghp_abcd1234</em>
+run.cmd --check --url https://github.com/octo-org/octo-repo --pat gcp_abcd1234
```
[the remark this answers is not in the document]
Comment (codewithdev) [reply content/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners.md]: Thanks for the suggestion. This absolutely works ✨

File: content/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry.md [hunk 4]
```diff
@@ -192,10 +192,10 @@ To use a published package from {% data variables.product.prodname_registry %},
   ```shell
   repositories {
       maven {
-          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else %}<em>REGISTRY-URL</em>{% endif %}/<em>OWNER</em>/<em>REPOSITORY</em>")
+          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else % REGISTRY-URL{% endif %}/OWNER/REPOSITORY")
```
Comment (ethanpalm) [inline content/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry.md:195]:
```suggestion
          url = uri("https://{% ifversion fpt or ghec %}maven.pkg.github.com{% else %}REGISTRY-URL{% endif %}/OWNER/REPOSITORY")
```

Comment (codewithdev) [reply content/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry.md]: Thanks @ethanpalm 👏🏻 
I literally missed that ;(

**Review by ethanpalm — APPROVED** [review]
I realized I accidentally dismissed my own review with that last commit 😅 Getting this up to date and merged ⚡

**Pull request merged**